packages/platform-browser/animations/async/src/async_animation_renderer.ts

```diff
@@ -78,10 +84,15 @@ export class AsyncAnimationRendererFactory implements RendererFactory2 {
       this._rendererFactoryPromise = this.loadImpl();
     }
 
-    this._rendererFactoryPromise?.then((animationRendererFactory) => {
-      const animationRenderer = animationRendererFactory.createRenderer(hostElement, rendererType);
-      dynamicRenderer.use(animationRenderer);
-    });
+    this._rendererFactoryPromise
+        ?.then((animationRendererFactory) => {
+          const animationRenderer =
+              animationRendererFactory.createRenderer(hostElement, rendererType);
+          dynamicRenderer.use(animationRenderer);
+        })
+        .catch(e => {
+          dynamicRenderer.handleRendererLoadingError();
```

Comment:
The delegation renderer should *not* be aware of the loading states (since it's unrelated to that abstraction). I think a better option would be to call `use` with an instance of the current renderer. In this case we permanently start using that fallback renderer. From the delegation renderer perspective, this operation is no different that applying animations renderer.
```suggestion
          // Permanently use regular renderer when loading fails.
          dynamicRenderer.use(<fallback_renderer_reference>);
```